insights/specs/datasources/lpstat.py

```diff
@@ -43,3 +45,45 @@ def lpstat_protocol_printers_info(broker):
     except Exception as e:
         raise SkipComponent("Unexpected exception:{e}".format(e=str(e)))
     raise SkipComponent
+
+
+@datasource(LocalSpecs.lpstat_o, HostContext)
+def lpstat_queued_jobs_number(broker):
+    """
+    This datasource provides the number of all queued job.
+
+    Typical content of ``/usr/bin/lpstat -o`` file is::
+
+        Cups-PDF-1802           root          265443328   Tue 05 Sep 2023 02:21:19 PM CST
+        Cups-PDF-1803           root          265443328   Tue 05 Sep 2023 02:21:21 PM CST
+        Cups-PDF-1804           root          265443328   Tue 05 Sep 2023 02:21:22 PM CST
+
+    Sample data returned::
+
+        3
+
+    Returns:
+        DatasourceProvider: Returns the collected content containing the number of all queued jobs.
+
+    Raises:
+        SkipComponent: When there is not any content.
+    """
+
+    content = broker[LocalSpecs.lpstat_o].content
+    if content:
+        cnt = 0
+        bad_lines = ["no such file or directory",
+                     "not a directory",
+                     "command not found",
+                     "no module named",
+                     "no files found for",
+                     "missing dependencies:",
+                     "Bad file descriptor"]
+        for line in content:
+            if not any(key in line for key in bad_lines):
+                cnt += 1
+        if cnt:
+            return DatasourceProvider(content="\n".join([str(cnt)]), relative_path='insights_commands/lpstat_-o_line_count')
```

Comment:
`"\n".join([str(cnt)])` == `"{0}".format(cnt)`

`"\n".join(['0'])` == `"0"`

Sorry for not pointing this out in the first review.

Reply:
`join` is not required